internal/api/hcpopenshiftcluster.go

```diff
@@ -93,10 +93,19 @@ func NewDefaultHCPOpenShiftCluster() *HCPOpenShiftCluster {
 			Type: arm.ManagedServiceIdentityTypeNone,
 		},
 		Properties: HCPOpenShiftClusterProperties{
+			Version: VersionProfile{
+				ChannelGroup: "stable",
+			},
```

Comment:
Do we want to set these if CS sets them already?

Reply:
I guess it would prevent breaking changes if CS changed the default for some reason.  I'm guessing that's what the response will be too.  If we define a default in our swagger, I'm cool with having a default in our code.

Reply:
I think we should.  These defaults are all explicitly named in the API spec.  They all currently agree with CS, but if CS were to change a default value we'd want the RP to favor the API spec.

Reply:
It did get me thinking, though, that if CS ever _did_ change a default and we wanted to match it in a new API version, I guess we'd have to make `NewDefaultHCPOpenShiftCluster` part of the versioned API interface since different API versions would now have different defaults.  We'll cross that bridge when we get there.